.circleci/config.yml

```diff
@@ -49,6 +49,17 @@ jobs:
            fingerprints:
              - "fb:f3:fe:be:1c:b2:ec:b6:25:f9:7b:a6:87:54:02:8c"
        - run: ci/api_mirror.sh
+   filter-example-mirror:
+     docker:
+       - image: *envoy-build-image
+     resource_class: xlarge
+     working_directory: /source
+     steps:
+       - checkout
+       - add_ssh_keys:
+           fingerprints:
+             - "fb:f3:fe:be:1c:b2:ec:b6:25:f9:7b:a6:87:54:02:8c"
```

Comment:
@mattklein123 do we need a new key for this?

Reply:
Yes I will make one and set it up for you.

Reply:
Use "f6:f9:df:90:9c:4b:5f:9c:f4:69:fd:42:94:ff:88:24"